advanced-client-install.rst

```diff
@@ -151,17 +151,21 @@ Installation
 ======================
 
 
-To install netmaker, you need a server token for a particular network, unless you're joining a network that allows manual signup, in which case you can join without a token, but the server will quarantine the machine until the admin approves it.
+To install netmaker, you need an enrollment key for a particular network, unless you're joining a network that allows manual signup, in which case you can join without an enrolment key, but the server will quarantine the machine until the admin approves it.
```

Comment:
To install netmaker?

Also we don't have any functionality of admin approval on server

Reply:
Pushed a commit with amendments.